### DISCUSSION TITLE
Disable confirmation pop-up when ending connection

### DISCUSSION THREAD

**Discussion by truthsword** [rustdesk/rustdesk#6091] [General]
When closing a connection, this appears...
![](https://i.imgur.com/6PDJj9e.png)

Please consider a per connection to disable this confirmation (or global disable).

Thanks!

**Comment by rustdesk** [comment]
@fufesou I mentioned this very earlier

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/6109 remove this since tv/anydesk does not have
